integration/spark/src/main/scala/org/apache/carbondata/indexserver/DistributedRDDUtils.scala

```diff
@@ -427,6 +427,6 @@ object DistributedRDDUtils {
       new java.util.ArrayList(),
       new java.util.ArrayList())
     new ExtendedBlockletWrapper(blocklets, request.getCarbonTable.getTablePath, request.getQueryId,
-      request.isWriteToFile, request.isCountStarJob)
+      request.isWriteToFile, request.isCountStarJob, request.getCdcVO)
```

Comment:
Refactor constructor of ExtendedBlockletWrapper to pass request directly, so that do not need to change the constructor everytime a new parameter is added

Reply:
refactored